persistence/src/main/java/rocks/metaldetector/persistence/domain/blogpost/BlogpostEntity.java

```diff
@@ -0,0 +1,38 @@
+package rocks.metaldetector.persistence.domain.blogpost;
+
+import lombok.*;
+
+import rocks.metaldetector.persistence.domain.BaseEntity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.validation.constraints.Size;
+
+@Getter
+@NoArgsConstructor
```

Comment:
This constructor does not have to be accessible from anywhere (an import is needed for this suggestion to work).
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED) // for hibernate and model mapper
```